Makefile

```diff
@@ -53,19 +54,39 @@ stylelint:
 unit-test:
 	npm run test:cov
 
+download-nodejs:
```

Comment:
Just `dnf install nodejs` inside all the containers that need it.

Reply:
That'll install node.js 8.x.x, not 6.9.x.

Reply:
adding the IRC log here b/c you dropped off just before that:

```
(12,50,56) atodorov: as part of the PR to minimize the number of docker files I propose to do `dnf install nodejs` which will pull 8.11 (current LTS) from Fedora 28
(12,51,10) atodorov: prior to that 6.9 has been specified in the files
(12,51,50) atodorov: I did the change and everything seems to build and pass tests. Is there any reason to continue using a soon to be unsupported version diretly from upstream instead of the version coming with Fedora 
(13,13,26) pitti: atodorov: what is "version directly from upstream" here? using the fedora version seems fine of course
(13,14,43) atodorov: pitti: all the docker files do curl https://nodejs.org/dist/***-tar.gz
(13,15,02) pitti: atodorov: oh, uh; I'd prefer the distro ones too indeed
(13,15,24) pitti: atodorov: maybe in the past that was done because fedora's packaged node was too old?
(13,15,27) pitti: but 8.11 certainly seems good
(13,15,44) atodorov: ok, any particular reason to specify 6.9? or was that historical ? 
(13,16,05) pitti: I don't have any knowledge about that; but if 8 works, that seems like a better choice if it's LTS indeed
(13,17,01) atodorov: yes it is, see https://github.com/nodejs/Release#release-schedule
```

Reply:
Reasonable to me! Thanks @atodorov.